CMakeLists.txt

```diff
@@ -101,6 +101,10 @@
 
 cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 
+if(POLICY CMP0148)
+    cmake_policy(SET CMP0148 OLD)
```

Comment:
We could probably move to https://cmake.org/cmake/help/latest/module/FindPython3.html#module:FindPython3 and use 3.12 as the minimum, right?

Reply:
Right. Easier for now :)